src/components/Header/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import BadgeIconButton from "./BadgeIconButton";
+import Button from "./Button";
+import Logo from "./Logo";
+import SearchBar from "./SearchBar";
+
+const Header = () => {
+  return (
+    <>
+      <Logo />
+      <SearchBar />
+      <div className="nav-section">
+        <BadgeIconButton
+          badgeIconButtonWrapper={"badge-icon-button-wrapper"}
+          badgeNumber={2}
+          icon={"far fa-heart"}
+          onClick={() => {
+            console.log("a");
+          }}
```

Comment:
delete console codes before pushing to github

Reply:
fixed